panther_driver/src/PantherKinematics.py

```diff
@@ -49,13 +50,13 @@ def inverseKinematics(self, wheel_FL_ang_vel, wheel_FR_ang_vel, wheel_RL_ang_vel
     def _getMotorSpeed(self, wheel_front_right, wheel_front_left, wheel_rear_right, wheel_rear_left):
```

Comment:
You may consider changing the function parameters here. What I mean is that I'd instead use a single parameter e.g. container object named "wheels_speed" and then operate with it in the method during the calculation. I'm not forcing here because the whole module uses this convention. Just consider this one.

Reply:
I don't think there is a point of doing that. The code needs refactor.